launch/docker.go

```diff
@@ -94,19 +99,41 @@ func (d *docker) runBuild(buildEntry buildEntry) error {
 	artVol := fmt.Sprintf("%s/:%s", hostArtDir, containerArtDir)
 	binVol := fmt.Sprintf("%s:%s", d.volume, "/opt/sd")
 	habVol := fmt.Sprintf("%s:%s", d.habVolume, "/opt/sd/hab")
+
+	// Overwrite steps for sd-local interact mode. The env will load later.
```

Comment:
[note] Discard all the steps defined in screwdriver.yaml.
I would implement a feature to easily call steps in [STEP2](https://github.com/screwdriver-cd/screwdriver/issues/2238#issuecomment-711872811).